src/main/java/ua/goit/gojava32/kickstarter/factory/DBHelper.java

```
@@ -0,0 +1,124 @@
+package ua.goit.gojava32.kickstarter.factory;
+
+import org.apache.log4j.Logger;
+import org.springframework.beans.factory.annotation.Autowired;
+import ua.goit.gojava32.kickstarter.dao.ImageDAOImpl;
+import ua.goit.gojava32.kickstarter.dao.ProjectDAOImpl;
+import ua.goit.gojava32.kickstarter.model.Image;
+import ua.goit.gojava32.kickstarter.model.Project;
+import ua.goit.gojava32.kickstarter.service.ImageService;
+import ua.goit.gojava32.kickstarter.service.ProjectService;
+
+import java.io.*;
+import java.sql.*;
+
+public class DBHelper {
+  public static final String CREATE_TABLES_FILENAME = "/sql/createTables.sql";
+  public static final String DROP_TABLES_FILENAME = "/sql/dropTables.sql";
+  public static final String INSERT_DATA_FILENAME = "/sql/insertData.sql";
+  public static final String TEST_IMAGE_FILENAME = "/images/medicine1.sql";
+
+  Connection connection;
+
+  public static void main(String[] args) {
+    DBHelper dbHelper = new DBHelper();
+    dbHelper.initDatabase();
+  }
+
+  public void initDatabase() {
+    final String CONNECTION_STRING = "jdbc:sqlite:kickstarter.db";
+    try {
+      connection = DriverManager.getConnection(CONNECTION_STRING);
+      executeQueryFromFile(DROP_TABLES_FILENAME);
+      executeQueryFromFile(CREATE_TABLES_FILENAME);
+      executeQueryFromFile(INSERT_DATA_FILENAME);
+      //saveImageToDatabase(TEST_IMAGE_FILENAME);
+    } catch (SQLException e) {
+      e.printStackTrace();
```

Swallowing exceptions is not good